core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/config/update/ConfigUpdateActions.java

```diff
@@ -0,0 +1,82 @@
+package com.sequenceiq.cloudbreak.core.flow2.cluster.config.update;
+
+import static com.sequenceiq.cloudbreak.core.flow2.cluster.config.update.ConfigUpdateEvent.CONFIG_UPDATE_FINALIZE_EVENT;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.AbstractClusterAction;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.ClusterViewContext;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.config.update.event.ConfigUpdateRequest;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.config.update.event.ConfigUpdateSuccess;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.provision.ClusterCreationEvent;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.provision.ClusterCreationState;
+import com.sequenceiq.cloudbreak.core.flow2.stack.AbstractStackFailureAction;
+import com.sequenceiq.cloudbreak.core.flow2.stack.StackContext;
+import com.sequenceiq.cloudbreak.core.flow2.stack.StackFailureContext;
+import com.sequenceiq.cloudbreak.core.flow2.stack.provision.action.AbstractStackCreationAction;
+import com.sequenceiq.cloudbreak.reactor.api.event.StackEvent;
+import com.sequenceiq.cloudbreak.reactor.api.event.StackFailureEvent;
+import java.util.Map;
+import javax.inject.Inject;
+import org.springframework.context.annotation.Bean;
+import org.springframework.statemachine.action.Action;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ConfigUpdateActions {
+
+    @Inject
+    private ConfigUpdateService configUpdateService;
+
+    @Bean(name = "CONFIG_UPDATE_START_STATE")
+    public Action<?, ?> configUpdateStartAction() {
+        return new AbstractStackCreationAction<>(StackEvent.class) {
```

Comment:
why not `AbstractClusterAction`?

Reply:
Cause I copied and altered and missed one :)

Reply:
Fixed